docs/security.md

```diff
@@ -88,17 +88,18 @@ The URI must use the hostname of server instead of IP.
-| `gravitino.authenticator.oauth.serviceAudience`   | The audience name when Gravitino uses OAuth as the authenticator.                                   | `GravitinoServer` | No                                         | 0.3.0         |
-| `gravitino.authenticator.oauth.allowSkewSecs`     | The JWT allows skew seconds when Gravitino uses OAuth as the authenticator.                         | `0`               | No                                         | 0.3.0         |
-| `gravitino.authenticator.oauth.defaultSignKey`    | The signing key of JWT when Gravitino uses OAuth as the authenticator.                              | (none)            | Yes if use `oauth` as the authenticator    | 0.3.0         |
-| `gravitino.authenticator.oauth.signAlgorithmType` | The signature algorithm when Gravitino uses OAuth as the authenticator.                             | `RS256`           | No                                         | 0.3.0         |
-| `gravitino.authenticator.oauth.serverUri`         | The URI of the default OAuth server.                                                                | (none)            | Yes if use `oauth` as the authenticator    | 0.3.0         |
-| `gravitino.authenticator.oauth.tokenPath`         | The path for token of the default OAuth server.                                                     | (none)            | Yes if use `oauth` as the authenticator    | 0.3.0         |
-| `gravitino.authenticator.kerberos.principal`      | Indicates the Kerberos principal to be used for HTTP endpoint. Principal should start with `HTTP/`. | (none)            | Yes if use `kerberos` as the authenticator | 0.4.0         |
-| `gravitino.authenticator.kerberos.keytab`         | Location of the keytab file with the credentials for the principal.                                 | (none)            | Yes if use `kerberos` as the authenticator | 0.4.0         |
+| Configuration item                                | Description                                                                                                                          | Default value     | Required                                   | Since version |
+|---------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|-------------------|--------------------------------------------|---------------|
+| `gravitino.authenticator`                         | It is deprecated since Gravitino 0.6.0. Please use `gravitino.authenticators` instead.                                               | `simple`          | No                                         | 0.3.0         |
+| `gravitino.authenticators`                        | The authenticators which Gravitino uses, setting as `simple`,`oauth` or `kerberos`. Multiple authenticators are separated by commas. | `simple`          | No                                         | 0.6.0         |
```

Comment:
You need to add the document to explain how resolve the header conflicts. For example, if two authenticators use the basic header.

Reply:
done. please help review it again

Reply:
@lw-yang 
Has this comment been fixed?